composer.json

```diff
@@ -12,16 +12,18 @@
         }
     ],
     "require": {
-        "php": ">= 7.1",
+        "php": ">= 8.1",
```

Comment:
Is this a breaking change?

Reply:
Yes, this PR drops support for versions less than 8.1 since they are at the end of their life cycle. Also GitHub Action only supports 8.1 and 8.2

Reply:
Alright, so this will be a major release then. Any other breaking changes that you that we should incorporate in the major release?

Reply:
There are no other breaking changes. I don't know how semver works on PHP but we should leave it as is. If the SDK doesn't work for 8.3 we will know immediately.

Reply:
I see your point, I made the change.

Reply:
Could you also take a look at https://github.com/parse-community/parse-php-sdk/blob/master/autoload.php? I'm not sire what that file is for, but contains line that references php 5.4

Reply:
That file is for when you don't want to use composer. I believe this package will work for PHP 5.4 or higher if you don't use composer.

Reply:
But we only test PHP 8.1 and 8.2 in the CI. How can we ensure that this package works with PHP 5.4?

Reply:
@mtrezza I've updated the autoload.php, is this good to go?

Reply:
Yes I think so, thanks!